overlord/devicestate/firstboot_test.go

```diff
@@ -429,10 +432,12 @@ func checkSeedTasks(c *C, tsAll []*state.TaskSet) {
 	c.Check(markSeededTask.WaitTasks(), testutil.Contains, otherTask)
 }
 
-func (s *firstBoot16Suite) makeSeedChange(c *C, st *state.State) *state.Change {
+func (s *firstBoot16BaseTest) makeSeedChange(c *C, st *state.State, opts *devicestate.PopulateStateFromSeedOptions, devAcct *asserts.Account,
+	checkTasks func(c *C, tsAll []*state.TaskSet), checkOrder func(c *C, tsAll []*state.TaskSet, snaps ...string)) *state.Change {
+
 	coreFname, kernelFname, gadgetFname := s.makeCoreSnaps(c, "")
 
-	s.WriteAssertions("developer.account", s.devAcct)
+	s.WriteAssertions("developer.account", devAcct)
```

Comment:
we should think how to avoid these changes, probably by having a *firstBootBaseTest on firstBoot16BaseTest, and a setupBaseTest that takes it

Reply:
something like this (obviously untested):

```
type firstBoot16BaseTest struct {
	*firstBootBaseTest
	// TestingSeed16 helps populating seeds (it provides
	// MakeAssertedSnap, WriteAssertions etc.) for tests.
	*seedtest.TestingSeed16
}

func (t *firstBoot16BaseTest) setup16BaseTest(c *C, bt *firstBootBaseTest) {
	t.firstBootBasetTest = bt
	t.setupBaseTest(c, &t.TestingSeed16.SeedSnaps)
}

func (s *firstBoot16Suite) SetUpTest(c *C) {
	s.TestingSeed16 = &seedtest.TestingSeed16{}

	s.setup16BaseTest(c, &s.fitstBootBaseTest)
	...
}
```

Reply:
Done.